### ISSUE TITLE
Custom Code no longer working in 0.7.1

### ISSUE THREAD

**Issue by jerry1970** [ramboxapp/community-edition#2460]
When I updated to 0.7.1 the Custom Code no longer seems to work. I had dark themes for WhatsApp and for Messenger but both are now back to their original brightlight!brightlight! themes. I understand that one of them might have changed their code so the CSS has the wrong selectors, but two at the same time?

### Steps to reproduce
1. upgrade to 0.7.1
2. WhatsApp and Messenger are 

### Environment
Happens on both Ubuntu and Windows. Below is the Windows info:
```
Version: 0.7.1
Platform: win32 (x64)
Electron: 7.1.1
Chromium: 78.0.3904.99
Node: 12.8.1
```

**Comment by freddyw** [comment]
confirmed, I have the exact same issue (Linux, Fedora 31 x64).

**Comment by saalkom** [comment]
Can confirm — I have userstyles to enable dark modes for Google Calendar, Facebook and FB Messenger; these worked under 0.6.9 (.deb) but no longer do so under 0.7.1 (snap). A userscript I have which redirects the FB newsfeed to "most recent" rather than "top stories" seems to work, so maybe only CSS is affected?

Xubuntu 19.10, x86-64

**Comment by fvulich** [comment]
We apologize for this inconvenience.
We already identify the error, and we are working on a fix.

**Comment by fvulich** [comment]
We released 0.7.2 which includes the fix for this issue.
